public_html/wp-content/themes/wordcamp-central-2012/style.css

```diff
@@ -1203,6 +1204,17 @@ li:after { content: "."; display: block; height: 0; clear: both; visibility: hid
 	padding-bottom: 20px;
 }
 
+/* !Featured Sponsor */
+.wordcamp-mes-sponsors > li {
+	width: 100%;
+	/*min-height: 300px;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;*/
+}
+.wordcamp-mes-sponsors .wordcamp-mes-sponsor__logo.align-center {
```

Comment:
I think this is redundant, since the blocks CSS is output on all sites. I see this style already applied with `.wordcamp-image__featured-image-container.align-center`.

Reply:
Good catch. be693910